modules/claims_api/app/controllers/claims_api/upstream_healthcheck_controller.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class UpstreamHealthcheckController < ::OkComputer::OkComputerController
+  end
+end
```

Comment:
this is purely for backwards compatibility with our current endpoint.

Reply:
the backwards compatibility is less for our outside consumers and more just because pingdom is already set up to hit this endpoint.